extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -483,12 +483,15 @@ export class AmpA4A extends AMP.BaseElement {
     }
     // If in localDev `type=fake` Ad specifies `force3p`, it will be forced
     // to go via 3p.
-    if (getMode().localDev &&
-        this.element.getAttribute('type') == 'fake' &&
-        this.element.getAttribute('force3p') == 'true') {
-      this.adUrl_ = this.getAdUrl();
-      this.adPromise_ = Promise.resolve();
-      return;
+    if (getMode().localDev && this.element.getAttribute('type') == 'fake') {
+      // Prevent issuing a second network request to try to fetch
+      // the same JSON again.
+      this.experimentalNonAmpCreativeRenderMethod_ = XORIGIN_MODE.NAMEFRAME;
```

Comment:
see my other comment. I'm trying to see if we can avoid doing this test only logic. We are making the test widget behave differently than the prod widget, so we're not testing the true logic.

If this does not work, we might be hitting a bug?